src/utils/usePreventDismiss.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import PreventDismissContext from './PreventDismissContext';
+
+type Options = {
+  enabled: boolean;
+};
+
+export default function usePreventDismiss({ enabled }: Options) {
+  const [symbol] = React.useState(() => Symbol(''));
```

Comment:
Shouldn't need an empty string here

Reply:
@kacperkapusciak I think that rule is not useful here. all of the symbols here will have the same name so there's no value provided by passing a name.